onnx/external_data_helper.py

```diff
@@ -41,8 +41,13 @@ def load_external_data_for_tensor(tensor, base_dir):  # type: (TensorProto, Text
     tensor: a TensorProto object.
     base_dir: directory that contains the external data.
     """
-    if tensor.HasField("raw_data"):  # already loaded
-        return
+    # If it has already loaded, override the raw_data from external tensor
+    if tensor.HasField("raw_data"):
+        # Empty raw_data for the following load
+        print("Warning: raw_data and external data exist simultaneously."
```

Comment:
not sure why we need this warning? This is confusing... we should never have both

Reply:
Why do we land in this situation? Maybe we are not clearing the raw_data field after serializing the raw data to external data... we should fix that

Reply:
After offline discussion, remove this warning because the override behavior is obvious when using load_external_data_for_tensor. Added a comment in the function instead